github3/orgs.py

```diff
@@ -509,6 +553,18 @@ def public_events(self, number=-1, etag=None):
         url = self._build_url('events', base_url=self._api)
         return self._iter(int(number), url, Event, etag=etag)
 
+    @requires_auth
+    def invitations(self, number=-1, etag=None):
+        r"""Iterate over outstanding invitations to this organization.
```

Comment:
Actually, it is a project convention to use raw strings for doc strings. I.e. I copied/pasted :)